CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # PayPal Android SDK Release Notes
 
+## unreleased
+* `PayPalNativePayments`: Bump NX0 to `0.8.8` and add `return_url`
```

Comment:
Looks like below in the changelog we use this syntax. Not sure we have established a preference, but we likely should so that we are able to stay consistent. 
```suggestion
* `PayPalNativePayments`:
  * Bump NX0 to `0.8.8` and add `return_url`
```

Reply:
We should also update `NX0` to `PayPal Native Checkout` since merchants won't know what `NXO` stands for